flink-streaming-java/src/test/java/org/apache/flink/streaming/api/operators/StreamTaskStateInitializerImplTest.java

```diff
@@ -192,7 +196,8 @@ public OperatorStateBackend createOperatorStateBackend(
 			streamOperator.getClass().getSimpleName(),
 			streamOperator,
 			typeSerializer,
-			closeableRegistry);
+			closeableRegistry,
+			UnregisteredMetricGroups.createUnregisteredOperatorMetricGroup());
```

Comment:
use `new UnregisteredMetricsGroup()` instead. (separate class, more generic than the ones from `UnregisteredMetricGroups`. Also applies to other usages.